overlord/fdestate/backend/reseal.go

```diff
@@ -127,15 +129,19 @@ func ResealKeyForBootChains(updateState StateUpdater, method device.SealingMetho
 	return nil
 }
 
-func resealRunObjectKeys(updateState StateUpdater, pbc boot.PredictableBootChains, authKeyFile string, roleToBlName map[bootloader.Role]string) error {
+func resealRunObjectKeys(updateState StateUpdater, pbcRunOnly, pbcWithRecovery boot.PredictableBootChains, authKeyFile string, roleToBlName map[bootloader.Role]string) error {
```

Comment:
let's keep in mind to maybe refactor this function and the resealRecoveryObjectKeys to pass a structure capturing most of the options.